andela/vale-ah-backend
# #164139761 Configure ESLint

**What does this PR do?**
- Configures ESLint to check for style violations (checking against Airbnb(_ish_) set of rules).
- Configures a pre-commit hook to lint and fix style violations
- Sets up Babel transpilation

**Description of Task to be completed?**
- Configure ESLint
- Set up pre-commit hooks to flag style violations
- Setup babel transpilation
- Fix npm vulns

**How should this be manually tested?**

-  Clone  the `ch-configure-eslint-164139761` branch locally
- Make and commit changes to any javascript source file

The linter will check committed files for style violations and attempt to automatically fix them before committing the changes

This PR also provisions a build script to transpile JavaScript source files to ES5
Run `npm run build` to test this

**Any background context you want to provide?**

**What are the relevant pivotal tracker stories?**
[#164139761](https://www.pivotaltracker.com/story/show/164139761)

**Screenshots (if appropriate)**
N/A

**Questions:**
N/A
